_posts/2023-03-01-release.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release 2023.09
+author: Priya
+date: 2023-03-01
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Warning for Duplicate Foreign View Key Columns in Join Rule
+Mammoth now warns the user when applying a join rule, only if the key columns from the foreign view used have non-unique values. This warning will be shown even if the rows don't exceed the threshold. If the rows exceed the threshold and result in a join runtime error, this warning will also be displayed.
+
+#### Data Deletion upon Deletion of Contributing Actions
```

Comment:
This is less of a feature but a defect fixed. So I would move it to that section and phrase it as follows

* When a Branch Out Task is suspended or deleted,  Mammoth deletes the contributed batch in the branched-out Dataset. This is also the behaviour now when a View containing such a Branch Out task is deleted or reset.